src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -912,6 +915,61 @@ OverlayImpl::crawl()
     return jv;
 }
 
+Json::Value
+OverlayImpl::crawl_server()
```

Comment:
nitpicking the name - this and the existing `crawl` seem poorly named. Probably should be `getCrawlInfo` or something like that. No action needed - just throwing it around.

Reply:
I'm down to change it `crawl -> getOverlayInfo` and `crawl_server -> getLocalServerInfo`

Reply:
Also was able to remove it from the `Overlay` interface since it's unused there